src/response.ts

```diff
@@ -34,24 +34,26 @@ import {
 
 import { GENERIC_STATUS, UNKNOWN } from './utils/TransactionStatesTypes';
 import { getConfigOrThrow } from './utils/config';
+import { WalletSession } from './sessionData/WalletSession';
+import { getOutcomeFromAuthcodeAndIsDirectAcquirer, OutcomeEnumType } from './utils/TransactionResultUtil';
 
 const config = getConfigOrThrow();
 
-const handleFinalStatusResult = (idStatus: GENERIC_STATUS, authorizationCode?: string) => {
-  const eventResult: string = getResultEventByAuthorizationCode(authorizationCode || '');
-  mixpanel.track(eventResult, {
-    EVENT_ID: eventResult,
+const handleFinalStatusResult = (idStatus: GENERIC_STATUS, authorizationCode?: string, isDirectAcquirer?: boolean) => {
+  const outcome: OutcomeEnumType = getOutcomeFromAuthcodeAndIsDirectAcquirer(authorizationCode, isDirectAcquirer);
+  mixpanel.track(PAYMENT_OUTCOME_CODE.value, {
```

Comment:
Reminder : after this change we'd update [mixpanel funnel](https://eu.mixpanel.com/project/2319753/view/2865553/app/dashboards#id=1023737) too